Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ruby:2.6.6-buster@sha256:1512719e1172fbcaa913997ffcb4b2be0948038fe255040a29f819f6c54c328d
```

Comment:
ISTR if you pin to a hash, that pins you to a specific architecture? We should check this works on both intel and ARM.